examples/informer/README.md

```diff
@@ -0,0 +1,41 @@
+# Informer Example
+
+Informers provide a high-level API for creating custom controllers for Kubernetes resources.
+
+This particular example demonstrates:
+
+* How to write an Informer against a core resource type and handle to create/update/delete events
+
+## Running
+
+``` go
+# assumes you have a working kubeconfig, not required if operating in-cluster 
+go run *.go -kubeconfig=$HOME/.kube/config
+```
+
+## Use Cases
+
+* Capturing resource events for logging to external systems (eg. monitor non-Normal Events and publish metrics to a time series database)
+* Creating lifecycle controllers for ThirdPartyResources (eg. coordinate create/update/delete of an external datastore represented via a ThirdPartyResource type)
+
+## Recommendations
+
+The design of `cache.NewInformer` lends itself towards writing simple, idempotent operations.
+It includes its own reconciliation loop which refreshes resources every resync period in addition to on-the-fly event handlers.
```

Comment:
Is there any sort of retry behavior in the `list/watch` if a handler on the Informer should fail?